sdcm/sct_config.py

```diff
@@ -1473,6 +1473,9 @@ class SCTConfiguration(dict):
              help="""Forces GossipingPropertyFileSnitch (regardless `endpoint_snitch`) to simulate racks.
              Provide number of racks to simulate."""),
 
+        dict(name="use_dns_names", env="SCT_USE_DNS_NAMES", type=boolean,
```

Comment:
I would suggest adding some verification step, to make sure we don't enable it for backend that are not tested yet.
or that we won't support for now, like k8s one ?